server/src/packages/pas-form/pas-form.service.ts

```diff
@@ -43,16 +43,24 @@ export class PasFormService {
           dcp_pasform
       `);
 
-      // if pasA is of a higher version than pasB, it should come first
-      const [{ dcp_pasform: latestPasForm }] = pasPackages.sort((pasA, pasB) => {
-        return pasB.dcp_packageversion - pasA.dcp_packageversion;
-      });
+      if (!pasPackages) {
```

Comment:
`if(pasPackages.length > 0)`
!!([]) actually evaluates to true

sidenote: handlebars makes it convenient for you to do this

Reply:
@NYCPlanning/engineering 
Updated and resolved!